docs/internal/RunDocumentationManjaro.md

```diff
@@ -0,0 +1,69 @@
+# Setting up MkDocs on Manjaro 
+
+This is a step-by-step guideline for installing and running the MkDocs Documentation on Manjaro Linux distribution.
+
+## MkDocs Installation
+
+As you may already know, MkDocs is Python-based. Therefore, you need to install a recent version of Python as well as its package manager, `Pip`. Luckily, both of these are installed by default on Manjaro (like on most Linux distros). If you are using the latest version of Python, then Pip is definitely installed by default.
+
+You may, however, need to upgrade Pip to latest version if it's not up-to-date.
+
+```
+pip install --upgrade pip
+```
+To check for the version of Python and Pip you are using:
+
+```
+python --version
+pip --version
+```
+You can now install MkDocs using Pip:
+
+```pip install mkdocs
```

Comment:
I think you need a newline after the triple ticks otherwise the code doesn't show.